src/frontend/main.go

```diff
@@ -186,7 +224,24 @@ func initTracing(log logrus.FieldLogger) {
 	// In a production environment or high QPS setup please use
 	// trace.ProbabilitySampler set at the desired probability.
 	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
-	initStackdriverTracing(log)
+	initOpenCensus(log)
+
+	// Initialize OpenTelemetry Tracer
```

Comment:
I don't think the comment is right. You are initializing the exporter to Cloud Tracing (Google product, former Stackdriver Tracing)

Reply:
sorry yeah, updated